ui/page/wallets/privacy_page.go

```diff
@@ -412,18 +415,19 @@ func (pg *PrivacyPage) showModalPasswordStartAccountMixer() {
 }
 
 func (pg *PrivacyPage) listenForMixerNotifications() {
+	if pg.AccountMixerNotificationListener == nil {
+		pg.AccountMixerNotificationListener = listeners.NewAccountMixerNotificationListener()
+	}
```

Comment:
`pg.AccountMixerNotificationListener` might not be nil but the channel might be closed, causing issues later when a channel send/read is attempted. Use the previously recommended partner of setting listeners to nil after closing their channels and returning if the listener is non-nil.